apps/happy-blocks/package.json

```diff
@@ -21,11 +21,14 @@
 	"dependencies": {
 		"@automattic/calypso-build": "workspace:^",
 		"@automattic/color-studio": "2.5.0",
+		"@automattic/format-currency": "workspace:^",
 		"@automattic/typography": "workspace:^",
 		"@wordpress/block-editor": "^9.1.0",
 		"@wordpress/blocks": "^11.8.0",
 		"@wordpress/components": "^19.15.0",
+		"@wordpress/element": "^4.7.0",
 		"@wordpress/i18n": "^4.9.0",
+		"lodash": "^4.17.21",
```

Comment:
Do you think we could avoid depending on lodash and use some native JS functionality instead?